### PR TITLE
867 Use only quotidian emojis in emoji field

### LINKED ISSUE #867: Use only quotidian emojis in emoji field
**Describe the bug**

Some of my more radical emoji choices might need reining in.  People on ancient Windows machines can't see

🫕 🪢 🫱🏿‍🫲🏽

They do all claim to have versions on Windows on [Emojipedia](https://emojipedia.org/) but I've proof it's lies, see:

image

We have already, long since, [enabled emojis](https://gohugo.io/quick-reference/emojis/) BUT we have not been using them because they only have white people, which would be extremely weird as a CYF brand. So I don't think that's a reasonable solution.

Probably just need to get a list of the least well supported ones and chenge them to something ~~more boring~~ more common.

### PR DESCRIPTION
**Pull request by SallyMcGrath**
## What does this change?

Add Noto emoji font and switch up the fallbacks. Will test on an ancient Windows machine tomoz

### Common Theme?

- [x] Yes

Fixes #867

## Checklist

- [x] I have read the [contributing guidelines](CONTRIBUTING.MD)
- [x] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [x] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [x] I have run my code to check it works
- [x] My changes follow our [Style Guide](https://curriculum.codeyourfuture.io/guides/code-style-guide)

### CODE REVIEW

**Comment by illicitonion** [comment]
This appears to have made all headings fall back to Times Roman for me on macOS ('Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/129.0.0.0 Safari/537.36') with no custom fonts installed locally, e.g. https://deploy-preview-1141--cyf-programming.netlify.app/overview/

Screenshot 2024-10-28 at 10 24 37

File: common-docs/hugo.toml [hunk 1]
```diff
@@ -44,7 +44,7 @@ title = "The longed for documentation"
         url = "https://codeyourfuture.io/donate"    
 
 [params]
-googleFonts="https://fonts.googleapis.com/css2?family=Oswald:wght@400;700&display=swap"
+googleFonts="https://fonts.googleapis.com/css2?family=Noto+Color+Emoji&Oswald:wght@400;700&display=swap"
```
Comment (SallyMcGrath) [inline common-docs/hugo.toml:47]:
```suggestion
googleFonts="https://fonts.googleapis.com/css2?family=Noto+Color+Emoji&family=Oswald:wght@400;700&display=swap"
```

File: org-cyf-theme/hugo.toml [hunk 2]
```diff
@@ -48,7 +48,7 @@ org = "https://github.com/CodeYourFuture"
 repo = "https://github.com/CodeYourFuture/curriculum/"
 pdrepo = "CYF-PD"
 root = "curriculum"
-googleFonts="https://fonts.googleapis.com/css2?family=Raleway:wght@300;600;800;900&family=Lato:wght@500;800&display=swap"
+googleFonts="https://fonts.googleapis.com/css2?family=Noto+Color+Emoji&Raleway:wght@300;600;800;900&family=Lato:wght@500;800&display=swap"
```
Comment (SallyMcGrath) [inline org-cyf-theme/hugo.toml:51]:
```suggestion
googleFonts="https://fonts.googleapis.com/css2?family=Noto+Color+Emoji&family=Raleway:wght@300;600;800;900&family=Lato:wght@500;800&display=swap"
```

**Review by SallyMcGrath — COMMENTED** [review]
Sally fatfingers woe

**Comment by SallyMcGrath** [comment]
image
image

Windows 10

**Pull request merged**